installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -0,0 +1,131 @@
+## Preparing the Kubernetes namespace
+The deploy directory referenced below can be found in [the manageiq-pods repo](https://github.com/ManageIQ/manageiq-pods/tree/master/manageiq-operator/deploy)
+
+1. Search for the Custom Resource Definition (CRD) and create it if it doesn't already exist.
+```bash
+oc get crds | grep manageiqs.manageiq.org
+```
+```bash
+oc create -f deploy/crds/manageiq.org_manageiqs_crd.yaml
+```
+
+2. Set up RBAC.
+```bash
+oc create -f deploy/role.yaml
+oc create -f deploy/role_binding.yaml
+oc create -f deploy/service_account.yaml
+```
+
+3. Deploy the operator in your namespace.
+```bash
+$ oc create -f deploy/operator.yaml
+```
+
+## Migrating from Appliances
+
+### Collect data from the appliance
+1. Take a backup of the database
+```bash
+pg_dump -Fc -d vmdb_production > /root/pg_dump
+```
+
+2. Export the encryption key and Base64 encode it for Openshift
+```bash
+vmdb && rails r "puts Base64.encode64(ManageIQ::Password.v2_key.to_s)"
+```
+
+3. Get the region number
+```bash
+vmdb && rails r "puts MiqRegion.my_region.region"
+```
+
+4. Get the GUID of the server that you want to run as.
+```bash
+vmdb && cat GUID
+```
```

Comment:
Perhaps not for this doc, but I wonder if we can have a rake task that just dumps these values..something like:

```bash
$ vmdb && rake evm:export:identity
dVhmSWdTQVVxNU96OGdvYy96SThIT09vMFNJKytTZDltZnBnQmFuWUlNND0=
0
252ca2e0-ebc9-0138-6e61-7affc243f4d3
```